tests/unit/test_registry.py

```
@@ -114,3 +117,180 @@ def test_import_shim_returns_registry_class() -> None:
 
     assert AtlascloudProvider is get_registry_provider_class("atlascloud")
     assert DeepAtlascloudProvider is AtlascloudProvider
+
+
+def test_registry_only_name_is_listed_as_supported(community_row: OpenAICompatibleProviderConfig) -> None:
```

Ruff (0.16.1)

[warning] Unused function argument: `community_row`

(ARG001)